src/functions.cpp

```diff
@@ -211,8 +212,8 @@ int32_t getFinalParameterValueLinear(int32_t paramNeutralValue, int32_t patchedV
 
 	// positivePatchedValue's range is ideally 0 ("0") to 1073741824 ("2"), but potentially up to 2147483647 ("4"). 536870912 represents "1".
 
-	return lshiftAndSaturate(multiply_32x32_rshift32(positivePatchedValue, paramNeutralValue),
-	                         3); // Must saturate, otherwise sustain level can easily overflow
+	return lshiftAndSaturate<3>(multiply_32x32_rshift32(
+	    positivePatchedValue, paramNeutralValue)); // Must saturate, otherwise sustain level can easily overflow
```

Comment:
nit: move the comment above so we get slightly better wrapping behavior here

Reply:
yeah that's fair